site/content/docs/commands/storage-init.en.md

```diff
@@ -3,17 +3,39 @@
 $ copilot storage init
 ```
 ## What does it do?
-`copilot storage init` creates a new storage resource attached to one of your workloads, accessible from inside your service container via a friendly environment variable. You can specify either *S3*, *DynamoDB* or *Aurora* as the resource type.
+`copilot storage init` creates a new storage resource as addons.
+
+By default, Copilot follows the "database-per-service" pattern:
+only the service or job that you specify during `storage init` will have access to that storage resource.
+The storage is accessible from inside the service's containers via a friendly environment variable.
```

Comment:
Maybe elaborate what the friendly env var is?

Reply:
changed to 

> only the service or job that you specify during `storage init` will have access to that storage resource.
The storage is accessible from inside the service's containers via a friendly environment variable, that
holds the name of your storage resource or credential information that is required is access the resource.